tasks.c

```diff
@@ -3506,6 +3506,9 @@ static BaseType_t prvCreateIdleTasks( void )
     TaskFunction_t pxIdleTaskFunction = NULL;
     BaseType_t xIdleTaskNameIndex;
 
+    /* Make sure that the below for loop doesn't go over the configIDLE_TASK_NAME macro. */
+    //configASSERT( sizeof( configIDLE_TASK_NAME ) >= configMAX_TASK_NAME_LEN );
```

Comment:
Why is this new assert commented with a `//`?

Reply:
Ah no, I was trying to figure out why are the checks failing.